components/general/icons/_icons.scss

```diff
@@ -0,0 +1,20 @@
+.icons { 
```

Comment:
Actually this one is necessary only for component preview purposes and should be added to `docs` directory.